### PR TITLE
Documentation

### LINKED ISSUE #56: Add documentation
We'll need to document the hooks we're adding and explain how to add custom post type and unpublished post support, since they're being done with filters. Also, the documentation needs to be formatted for WordPress. I have a couple useful scripts for this [here](https://github.com/mAAdhaTTah/wppb-mod/tree/master/bin).

### PR DESCRIPTION
**Pull request by mAAdhaTTah**
Closes #56. I still have to document the filters.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Changelog ##
```
Comment (benbalter) [inline CHANGELOG.md]: I usually prefer [releases](https://github.com/benbalter/wordpress-github-sync/releases) over maintaining a text-based changelog. That way Git/tags can do the heavy lifting for us. Not strongly opinionated if you're up for maintaining it. :smile:

File: .svnignore [hunk 2]
```diff
@@ -0,0 +1,11 @@
+bin
```
Comment (benbalter) [inline .svnignore]: Did you mean to add the `.svnignore` file?

File: CHANGELOG.md (hunk 1 above)
Comment (mAAdhaTTah) [inline CHANGELOG.md]: I plan on using the releases as well; we just need a text file for when we convert the README.md -> README.txt for deployment to WP.org. That wp2md script combines the CHANGELOG.md with the README.md file and then converts the whole thing to a WP.org readme. The other benefit is, according the Keep a Changelog standards, we can keep an Unreleased header which we update (and ask contributors to update) with each PR, which then just takes a few minor edits to convert into a release, so we don't have to write up the release each time, we just copy it over.

That said, you did get me thinking about whether it's possible to pull down from the API and convert that to the CHANGELOG, which would be really cool, but we're really close to a 1.0.0, I think, so maybe I'll figure that out and add it in the future.

File: .svnignore (hunk 2 above)
Comment (mAAdhaTTah) [inline .svnignore]: Self-cleaning .svnignore :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (benbalter) [inline CHANGELOG.md]: >  I think, so maybe I'll figure that out and add it in the future.

Makes total sense. :+1:

**Comment by mAAdhaTTah** [comment]
@benbalter Enable the wiki? I don't think we need to document all the filters on WP.org.

**Comment by benbalter** [comment]
> Enable the wiki? 

Done. :smile:

**Comment by mAAdhaTTah** [comment]
I updated the wiki with some basic info about the filters added as well.

**Pull request merged**